gmacpyutil/gmacpyutil/cocoadialog.py

```diff
@@ -326,6 +319,96 @@ def GenerateCommand(self):
     return super_cmds
 
 
+class OK_MsgBox(TweakDialog):
```

Comment:
Yes it would 👍
I've disabled the Get/SetButton[x] methods as they are not used in these cocoadialog run modes.